src/test/java/core/basesyntax/WorkWithFileTest.java

```diff
@@ -61,8 +72,11 @@
 
     @Test
     public void getStatisticAboutBanana() {
+        try {
         workWithFile.getStatistic("banana.csv", BANANA_RESULT_FILE);
-
+        } catch (IOException e) {
+            System.out.println("Can't write data to file");
+        }
```

Comment:
The try-catch block for `IOException` is not necessary in test methods. Letting the exception propagate will cause the test to fail and provide more detailed information about the failure.